BUILDER_DEV.md

```diff
@@ -1,221 +1,67 @@
+1. Set the value of `User authorization callback URL` to `http://${APP_HOSTNAME}/` (The trailing `/` is *important*)
+1. Set the value of `Webhook URL` to `http://${APP_HOSTNAME}/`
+1. Set everything to read only (this is only used for your org so it's safe)
+1. Save and download the pem key
+1. Copy the pem key to `${HABITAT_SRC_ROOT}/.secrets/builder-dev-app.pem`
+1. Record the the client-id, client-secret and app_id. These will be used for the `GITHUB_CLIENT_ID`, `GITHUB_CLIENT_SECRET` and `GITHUP_APP_ID` build variables (seen below).
 
-* Run the provision script
-
-```
-$ ./support/linux/provision.sh
-```
+### Setup GitHub Teams
 
-* Source cargo
+1. Create a GitHub team in your organization for administrators and retrieve the team-id. This will be used for the value of `GITHUB_ADMIN_TEAM`
 
-```
-$ source $HOME/.cargo/env
-```
+> note: The only way to retrieve the ID for a GitHub team is currently [by the API](https://github.com/rei/github-api-utils/blob/master/bash/get-team-id.sh)
 
-* Run the build-srv makefile (NOTE - if you receive an error about cargo not being found, make sure you have removed the "secure_path" section of /etc/sudoers)
+### Studio requirements
 
-```
-$ make build-srv
-```
+1. Ensure you have run `hab setup` in your environment that will be executing the studio
 
-* Run this command (it may take awhile) to start all the Builder services
+### Starting the services
 
-```
-$ sudo -E make bldr-run-no-build
-```
+From either your VM or Mac:
 
-* Wait for all services to start - they are started when you see worker heartbeat debug entries in the log. If services do not start because some were already running from a previous attempt, run `$ make bldr-kill`
+* `cd <habitat source path>`
+* `direnv allow`
+* `hab studio enter`
+* `start-builder`
 
 ### UI Setup
 
+You should only need to do this if you are developing the UI
+
 * Follow the instructions in the [Web UI README](https://github.com/habitat-sh/habitat/blob/master/components/builder-web/README.md) to get the Web UI running locally.
 * Open up a browser and navigate to http://localhost:3000/#/pkgs - you should see the Habitat UI running locally.
 
-### Creating an Origin
-
-### Option A: Create an origin using the web UI
-
-* Navigate to http://localhost:3000/#/pkgs
-* Log in
-* You may need to accept the Habitat Builder Dev Githhub app in your Github account.
-* Click on "Create New Origin"
-* Fill out form (call the origin "core"), click "Save & Continue"
-
-### Option B: Create an origin using the command line
-
-1. Create an origin in the DB by issuing the following command:
-
-```
-$ http POST http://localhost:9636/v1/depot/origins Content-Type:application/json Authorization:Bearer:${HAB_AUTH_TOKEN} name=${HAB_ORIGIN}
-```
-
-The response should be something like:
-
-```
-HTTP/1.1 201 Created
-
-{
-"id": 151409091187589122,
-"name": "core",
-"owner_id": "133508078967455744",
-"private_key_name": ""
-}
-```
-### Install Dependencies in your local Builder Env
```

Comment:
I think this section needs to stay or be rewritten since the `Run a build` section is still around and references these directions for installing/uploading packages from production builder.